src/network/src/network.cpp

```diff
@@ -1,12 +1,12 @@
-#include "network.hpp"
+#include "network/network.hpp"
```

Comment:
`src/network/network.cpp → src/network/src/network.cpp` I do not understand why we need one more time `src` there, why not  `src/network/network.cpp`

Reply:
And also if there is reason for that why files like `src/network/rpc/WSServer.cpp` are not part of `src`

Reply:
My further plan is to rename get rid of root src and move all that libs to libraries dir as Jakub suggested above. But I planned to make it in a separate PR right after it to discuss it separately. So every lib will have its own src and there will be no mess in library root dir. You will see CMakeLists.txt, src and include directories in every lib dir

Reply:
why not to do it straight away ?